### DISCUSSION TITLE
I wish there was an achievement for committing for 3 consecutive days.

### DISCUSSION THREAD

**Discussion by Suletta-Majo** [community/community#120738] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It's been a while since I got the Pullshark achievement, but there's nothing else and I miss it.  
I thought it happy to have something like a subject line, so I'll give you some feedback.   

❤️(it for get Heart On Your Sleeve achievement)

omg  
I just found this announcement and am reading it now,  
more spam to get achievements... so my plan was not so good 🙄
##### Achievements will no longer be available in the Community  
https://github.com/orgs/community/discussions/106536

**Comment by ebndev** [comment]
Hi @Suletta-Majo,

Thank you for being a part of the GitHub community and reading the announcement.

I do want to reiterate that posts and comments geared towards unlocking achievements will lead to being blocked.

We want to make sure there is space for users to ask questions without overwhelming them with other conversations so I will be closing this discussion. Thank you for helping us maintain a productive and tidy community for all our members.
